.circleci/config.yml

```diff
@@ -52,12 +53,10 @@ aliases:
 
   test_template: &test_template
     steps:
-      - checkout
-      - run: ./scripts/assert-changed-files.sh "packages/*|.circleci/*"
-      - <<: *restore_cache
-      - <<: *install_node_modules
-      - <<: *persist_cache
       - <<: *attach_to_bootstrap
```

Comment:
Cool, thought we always had to do a yarn install to get the symlinks installed correctly. I guess circleci takes care of this! :tada:

Reply:
There is one task that I had to leave the install_node_modules in. It seems like some sharp npm has binaries, and that was having issues between jobs.